STMReco/src/PlotSTMWaveformDigis_module.cc

```diff
@@ -39,12 +43,18 @@ namespace mu2e {
     void analyze(const art::Event& e) override;
 
     art::InputTag _stmWaveformDigisTag;
+    bool _subtractPedestal;
+    int _verbosityLevel;
+    ProditionsHandle<STMEnergyCalib> _stmEnergyCalib_h;
     STMChannel _channel;
   };
 
   PlotSTMWaveformDigis::PlotSTMWaveformDigis(const Parameters& config )  :
     art::EDAnalyzer{config},
-    _stmWaveformDigisTag(config().stmWaveformDigisTag())
+    _stmWaveformDigisTag(config().stmWaveformDigisTag()),
+    _subtractPedestal(config().subtractPedestal()),
+    _verbosityLevel(config().verbosityLevel()),
+    _stmEnergyCalib_h()
```

Comment:
unnecessary, but harmless, I'd remove it

Reply:
Removed here [99278c1](https://github.com/Mu2e/Offline/pull/879/commits/99278c169ad096e3da1101df0b90d723c419d904)